### DISCUSSION TITLE
M1 support

### DISCUSSION THREAD

**Discussion by aditya71** [pilotmoon/Scroll-Reverser#118] [Q&A] [answered]
Hey, I have loved using this for my mac, it makes my life so much easier. I am going to upgrade to the new m1 mac soon and I was wondering if there was or (going to be) native apple silicon support. I can't imagine using a mac without this app. Thanks.

**Comment by aditya71** [comment, marked as answer]
Never mind, found my answer on [the website](https://pilotmoon.com/scrollreverser/) in the Q/A section.

**Reply by pilotmoon** [reply]
I've updated the repo's README to mention M1 support too. Thanks for the feedback and kind words 👍
